### DISCUSSION TITLE
Smite Futures

### DISCUSSION THREAD

**Discussion by Alisyntx** [thatskymod/Sky-CotL-Scripts#42] [Ideas]
It's possible to add in smite?  the Hide buttons controllers in sky ? So its much better when we are recording using Android..

**Comment by thatskymod** [comment]
@Alialialiu,

Yes, it **is** generally possible to add an on/off toggle for the UI elements (controls and icons) in libsmite. I recognize and sympathize with your insight. However, as a repository maintainer, I cannot guarantee its implementation or provide a specific timeline. For the latest information, consider reaching out to [@gxosty](https://github.com/gxosty) via their designated Discord [server](https://discord.com/invite/fDFU5QWKgh).

In the meantime, you might consider exploring other scripts that offer this feature.
